partcad/src/partcad/converters/base_converter.py

```
@@ -0,0 +1,16 @@
+from pathlib import Path
+from OCP.TopoDS import TopoDS_Shape
+
+
+class BaseConverter:
+    def __init__(self, input_file: Path, output_file: Path):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)